enforcer/enforcer.go

```diff
@@ -13,8 +12,9 @@ import (
 
 // Config configuration for initializing casbin Enforcer
 type Config struct {
-	Model  string
-	Policy string
+	Model    string
+	File     string
```

Comment:
The name `Policy` is much better than `File`.
What is `File` stand for? Config file? log file?